src/containers/HomeContainers/Home.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React, { useState, useEffect } from 'react'
+import { Text, FlatList, View } from 'react-native'
+import { getStarredOfferingsData } from '../../api/offerings'
+import CourseCard from '../../components/Cards/CourseCard'
+import styles from './Home.style'
+
+function fetchCourseInfo(setCourses) {
+  getStarredOfferingsData()
+    .then((response) => {
+      console.log(response)
+      return response
+    })
+    .then((data) => setCourses(data))
+}
+
+const renderCourseItem = ({ item }) => {
+  const info = {
+    courseId: item.courses[0].courseId,
```

Comment:
We should have some check here to see if `item.courses` is empty to avoid crashing